client/src/components/SubmitButton.js

```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled';
+
+const SubmitButton = styled.button`
+  width: 136px;
```

Comment:
Have a look at the comments regarding fixed widths on this PR: 
https://github.com/florianGierlichs/mycrypt/pull/24

> But I don't think you should set the height and width at all. It should be calculated automatically by the content and font-size.
If you have a button designed, take a look at the padding and font-size. This is all you need.
If you set a fixed width, the button would have issues with long content.

Reply:
Thank you for the heads up! I will consider this in the other components. However this button only has two states with same size. So it should be fine for this component.